locksmith/src/utils/middlewares/cacheMiddleware.ts

```diff
@@ -26,6 +26,12 @@ export const createCacheMiddleware = (option: Partial<Options> = {}) => {
     if (req.method !== 'GET') {
       return next()
     }
+
+    // We don't cache authenticated requests
+    if (req.user?.walletAddress) {
```

Comment:
I think that would not apply to `app` users? Can we just make it if `req.user` is defined?

Reply:
It does apply to app users.